cli/cli-client/src/main/java/org/infinispan/cli/shell/Completer.java

```diff
@@ -21,7 +23,7 @@ public Completer(Context context) {
    @Override
    public void complete(CompleteOperation op) {
       String buffer = op.getBuffer();
-      List<String> candidates = op.getCompletionCandidates();
+      List<String> candidates = op.getCompletionCandidates().stream().map(TerminalString::toString).collect(Collectors.toList());
```

Comment:
2 concerns:
* I'm not sure if ```toString``` is correct here. It seems to apply formatting options (color) to the string. I think ```getCharacters``` is more correct.
* ```CompleteOperation``` has a method to add candidates (```addCompletionCandidate(String)```), so it can avoid to stream and collect them.